scripts/deploy_llama.py

```diff
@@ -4,240 +4,240 @@
 def main():
     acct = accounts.load("llama_deployer")
     premint_addrs = [
```

Comment:
it's missing my address for the premint :cry: 

the WLs can own multiple Llamas? lucky peeps